src/scss/jooa11y.scss

```diff
@@ -428,28 +439,33 @@ html[data-jooa11y-theme="dark"] {
 }
 
 #jooa11y-container .jooa11y-warning-badge {
-  color: var(--jooa11y-panel-bg);
+  color: var(--jooa11y-panel-bg-warning);
   background: var(--jooa11y-yellow-text);
 }
 
 #jooa11y-container #jooa11y-outline-list .jooa11y-outline-2 {
-  margin-inline-start: 15px;
+  -webkit-margin-start: 15px;
+          margin-inline-start: 15px;
 }
 
 #jooa11y-container #jooa11y-outline-list .jooa11y-outline-3 {
-  margin-inline-start: 30px;
+  -webkit-margin-start: 30px;
+          margin-inline-start: 30px;
 }
 
 #jooa11y-container #jooa11y-outline-list .jooa11y-outline-4 {
-  margin-inline-start: 45px;
+  -webkit-margin-start: 45px;
+          margin-inline-start: 45px;
 }
 
 #jooa11y-container #jooa11y-outline-list .jooa11y-outline-5 {
-  margin-inline-start: 60px;
+  -webkit-margin-start: 60px;
+          margin-inline-start: 60px;
 }
 
 #jooa11y-container #jooa11y-outline-list .jooa11y-outline-6 {
-  margin-inline-start: 75px;
+  -webkit-margin-start: 75px;
```

Comment:
why are you adding all these webkit prefixed classes to the scss there is no need for it at the margin-inline-start is supported by all browsers